hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/MockSCMHADBTransactionBuffer.java

```diff
@@ -20,22 +20,22 @@
 import org.apache.hadoop.hdds.utils.db.BatchOperation;
 import org.apache.hadoop.hdds.utils.db.DBStore;
 import org.apache.hadoop.hdds.utils.db.RDBBatchOperation;
+import org.apache.hadoop.hdds.utils.db.Table;
 import org.apache.ratis.statemachine.SnapshotInfo;
 
 import java.io.IOException;
 
-public class MockDBTransactionBuffer implements DBTransactionBuffer {
+public class MockSCMHADBTransactionBuffer implements SCMHADBTransactionBuffer {
   private DBStore dbStore;
   private BatchOperation currentBatchOperation;
 
-  public MockDBTransactionBuffer() {
+  public MockSCMHADBTransactionBuffer() {
   }
 
-  public MockDBTransactionBuffer(DBStore store) {
+  public MockSCMHADBTransactionBuffer(DBStore store) {
     this.dbStore = store;
   }
 
-  @Override
   public BatchOperation getCurrentBatchOperation() {
```

Comment:
change getCurrentBatchOperation  to private.

Reply:
+1